js/client-battle-tooltips.js

```diff
@@ -408,6 +425,25 @@ var BattleTooltips = (function () {
 		text += '</div></div>';
 		return text;
 	};
+	
+	BattleTooltips.prototype.findBoost = function (pokemon, stat) {
```

Comment:
You might want to rename this `calculateBoostedStat`, pass in `myPokemon.stats`, and pass out a stat object of all the stats.

You might also want to add other stat boosts, such as Scarf, Unburden, Pure Power, etc.

You might also want to show after boosts only if the Pokémon is boosted in some way. Perhaps even bold the boosted stats.